IRS 90918-10 API/openAPI/v2/api/FSM_MVP_Specs.yml

```diff
@@ -9,7 +9,6 @@ info:
     Specifications for the OSDM API standard. The OSDM specification supports two
     modes of operation: Distributor Mode and Allocator Mode. The API works the same in 
     both mode, except that in allocator mode the API also returns fare information.
-    
     The OSDM specification combines the nTM as well as the FSM initiative into one aligned
```

Comment:
the abbreviations FSM and nTM will disappear, we should refere to the allocator mode and the distributor mode

Reply:
OK. Fixed with next pull request